### PR TITLE
Issue#1595

### PR DESCRIPTION
**Pull request by TedaLIEz**
I add a link to in-app faq in the backup UI according to the Issue#1595
<a href="https://github.com/open-keychain/open-keychain/issues/1595">https://github.com/open-keychain/open-keychain/issues/1595</a>

### CODE REVIEW

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java [hunk 1]
```diff
@@ -144,10 +144,12 @@ static void brandGlowEffect(Context context, int brandColor) {
                 //glow
                 int glowDrawableId = context.getResources().getIdentifier("overscroll_glow", "drawable", "android");
                 Drawable androidGlow = context.getResources().getDrawable(glowDrawableId);
+                assert androidGlow != null;
```
Comment (dschuermann) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java]: why has this been added?

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java [hunk 2]
```diff
@@ -99,4 +104,14 @@ public void onClick(View v) {
     protected void initLayout() {
         setContentView(R.layout.help_activity);
     }
+
+    @IntDef({TAB_START, TAB_FAQ, TAB_CONFIRM, TAB_CHANGELOG, TAB_ABOUT})
+    @Retention(RetentionPolicy.SOURCE)
+    private @interface startHelpActivity {}
```
Comment (dschuermann) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java]: Why are these annotations here + the interface?

Comment (TedaLIEz) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java]: I want to make sure that param code in the method should be one of {TAB_START, TAB_FAQ, TAB_CONFIRM, TAB_CHANGELOG, TAB_ABOUT}(Just for check), I don't know whether this is acceptable.

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java (hunk 1 above)
Comment (TedaLIEz) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java]: There is a lint in my androidstudio before I added it.

Comment (dschuermann) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/KeychainApplication.java]: Please don't follow lint suggestions blindly. Remove these changes.

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java (hunk 2 above)
Comment (dschuermann) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java]: Much overhead just to check this. PLease remove this.

**Comment by dschuermann** [comment]
Please implement my suggestions and do a git squash, it should be only one commit.

**Comment by TedaLIEz** [comment]
I made a git squash just now and remove changes you mentioned:)

**Comment by dschuermann** [comment]
Thanks for your help, I merged your changes and updated the FAQ to include the information how to import.

**Pull request merged**